Adding a disabling button feature with new css style

PabloFLPs opened the pull request:
Just adding a feature to disable button on form submitting (and also, enabling it again if the promise results in an error). Also, I needed the previous PR, but as it have only a few changes, I opted to add them here and make another PR; this way, I would not need you to approve the other one to make these changes.

PabloFLPs committed:
Adding a disabling button feature with new css style

PabloFLPs commented:
Oh, other thing: I preferred to keep the button disabled after the form submitting (if it is succesful) and not only while tht Heroku is receiving the data. But if you prefer the way you sugested, I can change any time.

Erickinhou commented on src/components/Contact.tsx:
```diff
@@ -46,10 +48,15 @@ export default function Contact() {
+    } catch (error) {
       if(error) errorToastEmitter()
-    })
+      setDisableButtonState(false)
```
The setDisableButtonState needs to be out of the `try/catch` statement.

PabloFLPs replied:
Well, I don't like this way, but ok. I will makes the changes here.

Erickinhou commented on src/styles/modules/_contact.module.scss:
```diff
@@ -9,11 +9,17 @@
-}
\ No newline at end of file
+  
+  button:disabled {
```
Cool

Erickinhou commented:
> Oh, other thing: I preferred to keep the button disabled after the form submitting (if it is succesful) and not only while tht Heroku is receiving the data. But if you prefer the way you sugested, I can change any time.

If the user accidentally sends the message, does he need to reload the page?

Erickinhou reviewed (changes requested):
I hit the submit button and the button is still disabled. If the user accidentally sends the message, does he need to reload the page? I don't think this is a good idea.
![image](https://user-images.githubusercontent.com/57305093/145810767-135ab40c-cb6f-4739-8746-2fd29087e1e5.png)

PabloFLPs committed:
updatingDisablingButton

Just changing the button disabling 'setDisableButtonState(false)' to outside the 'try/catch' block.

PabloFLPs committed:
Merge branch 'master' into disabledButton
